lib/thesis/views/api_view.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Thesis.ApiView do
+  use Thesis.View, :view
+
+
+  def render("page.json", %{thesis_page: page} = assigns) do
+    %{
+      id: page.id,
+      slug: page.slug,
+      title: page.title,
+      redirect_url: page.redirect_url,
+      page_contents: render("page_contents.json", assigns[:page_contents]),
+      inserted_at: page.inserted_at,
+      updated_at: page.updated_at
+    }
+  end
```

Comment:
It looks like it -- note that the javascript only makes use of the `slug` in the response currently, at any rate.